About.html

```diff
@@ -0,0 +1,143 @@
+        </div>
+    </div>
+    <section class="sidebar">
+        <div class="sidebar-inner">
+            <div class="sidebar-cancel">
+                <button class="cancel" type="button">&times;</button>
+            </div>
+            <div class="sidebar-menu">
+                <a href="index.html" class="anchor-link">Home</a>
+                <a href="about.html" class="anchor-link active">About</a>
+                <a href="#program" class="anchor-link">Program</a>
+            </div>
+        </div>
+    </section>
+    <section class="hero container-fluid py-5 shadow">
+        <div class="hero-inner text-center">
+            <h4 class="display-hello d-block d-md-none">"Hello! Future World"</h4>
+            <div class="mx-0">
+                <h1 class="text-uppercase display-title">Robotics Team Confrance 2022</h1>
+            </div>
+            <div class="para-container-about bg-light">
+                <p class="display-paragrapg">The Robotics Team Confrance brings together the community of experts, academics and
+                    activists who comprise the Robotics network in a different country.
+                    This year, we hope to expand our invitation list including orgranizations and individuals.
+                </p>
+            </div>
+            <div class="text-center my-3">
+                <p>Please contact us per email for any further questions about Robotics Team Confrance 2022!</p>
+                <a href="www.robotics@team.org" class="text-dark">robotics@team.org</a>
+                <span><i class="fa-regular fa-angle-right mx-2"></i></span>
+                
+            </div>
+        </div>
+
+    </section>
+    <section class="mt-5 shadow  p-4">
+        <h4 class="text-center my-4">Creative Robotics Team Confrance Logo 2022</h4>
+        <p class="mx-5 text-center p-3">The logo of Robotics Confrance Logo 2022 was decided through the logo competition from 8. June to 7. July.
+            After online public votingand selection committees judging process. The logo of Ahmad won the prize;
+            After consultation with Mr.Ahmad we optimized the prize wining logo for this years summit as following.
+        </p>
+        <div class="d-flex justify-content-center pb-5">
+            <img src="images/logo2.jpg" alt="2015 cc global summit logo ">
+        </div>
+    </section>
+    <section class="mt-5 mb-4 shadow">
+        <h4 class="text-center my-4">See the past Robotic Confrances</h4>
+        <p class="text-center">Take a look at last two Robotic Confrance which take place in Kabul, Afghanistan</p>
+        <div class="row d-flex justify-content-center mx-5 py-4">
+            <div class="col-12 col-md-6 d-flex justify-content-center ">
+                <div class="hasHover">
+                    <img src="images/Robotics_Summit.jpeg"
+                        alt="image of global summit 2013">
+                    <div class="abso"></div>
+                </div>
+
+            </div>
```

Comment:
- [ ] I would suggest you put some spaces between the two images, to make it better and stick to the provided design

Your design:
![image](https://user-images.githubusercontent.com/80612925/175429057-b3dbccaf-68bc-4260-b481-1ef89845bc92.png)

Template design:
![image](https://user-images.githubusercontent.com/80612925/175429090-6b43ce0a-932e-4fcf-90fc-996520b204b3.png)